style.css

```diff
@@ -0,0 +1,101 @@
+
+body {
+  font-family: "Poppins", sans-serif;
+  padding: 0%;
+  margin: 0;
+}
+
+header {
+  height: auto;
+  width: 100%;
+  border-radius: 0.25rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+}
+
+.my-logo {
+  width: fit-content;
+  font-style: normal;
+  color: #6070ff;
+  font-size: 18px;
+  line-height: 20px;
+  text-align: center;
+  font-weight: 700;
+  margin: 10px 12px;
+  text-decoration: none;
+}
+
+.nav-bar {
+  width: 18px;
+  height: 12px;
+}
+
+.menu-buttom {
+  margin: 6px 16px;
+  text-align: center;
+}
+
+section {
+  background: url(images/header-shapes\ mobile@2x.svg);
+  height: 100vh;
+  background-repeat: no-repeat;
+  background-size: 100% auto;
+  padding: 0 20px;
+}
```

Comment:
- [ ] As the previous reviewer suggested, and according to the requirements please consider using flexbox to layout the elements of each section:

![image](https://user-images.githubusercontent.com/77069011/219541513-a4dd94b6-7c70-4c33-973c-a4f7bf07827b.png)

Flexbox gives you a lot of options to build a great responsive layout, please use it! 🤓 🔧